uberenv.py

```diff
@@ -736,7 +742,8 @@ def patch(self):
         self.print_spack_python_info()
 
         # force spack to use only "defaults" config scope
-        self.disable_spack_config_scopes(spack_dir)
+        os.environ["SPACK_DISABLE_LOCAL_CONFIG"] = "1"
```

Comment:
For the records, I think this is not what we want:
With this, the config scope is disabled only in the script context. That means the uberenv spack instance can get "polluted" by an out of script use.

Reply:
This PR is fairly out of date with develop, it looks like uberenv is currently implementing this technique in develop for newer versions of spack.